pages/osx/launchd.md

```diff
@@ -0,0 +1,4 @@
+# launchd
+
+> This manages processes, both for the system and users.
+> You cannot invoke launchd manually.
```

Comment:
No examples? even a help page?

Reply:
I think we should have something like

```
- Run init:

`launchd`
```

maybe even

```
- Run init:

`/sbin/launchd`
```

since it is only invoked this way by the kernel

Reply:
oops, I forgot to come back to this one.  thanks for catching this.  Is it allowed to have multiple links for more information?  there are several good pages with in-depth info about launchd:
* https://support.apple.com/guide/terminal/script-management-with-launchd-apdc6c1077b-5d5d-4d35-9c19-60f2397b2369/mac
* https://www.launchd.info
* https://en.wikipedia.org/wiki/Launchd
I added the first link to the page, but the other links would probably be helpful too.